reference/3.0/Microsoft.PowerShell.Core/About/About.md

```diff
@@ -17,445 +17,337 @@ About topics cover a ranges of concepts about PowerShell.
 
 ### [about_Do](about_Do.md)
-
 Runs a statement list one or more times, subject to a While or Until condition.
 
 ### [about_Environment_Provider](about_Environment_Provider.md)
-
-Provides access to the Environment variables and the values that they represent.
+Describes the features of the Environment provider.
 
 ### [about_Environment_Variables](about_Environment_Variables.md)
-
-Describes how to access Windows environment variables in Windows PowerShell.
+Describes how to access Windows environment variables in PowerShell.
 
 ### [about_Eventlogs](about_Eventlogs.md)
-
 Windows PowerShell creates a Windows event log that is named "Windows PowerShell" to record Windows PowerShell events.
 
 ### [about_Execution_Policies](about_Execution_Policies.md)
-
 Describes the PowerShell execution policies and explains how to manage them.
 
 ### [about_FileSystem_Provider](about_FileSystem_Provider.md)
-
-Provides access to the FileSystem objects.
+Describes the features of the FileSystem provider.
 
 ### [about_For](about_For.md)
-
 Describes a language command you can use to run statements based on a conditional test.
 
 ### [about_Foreach](about_Foreach.md)
-
 Describes a language command you can use to traverse all the items in a collection of items.
 
 ### [about_Format.ps1xml](about_Format.ps1xml.md)
-
-The `Format.ps1xml` files in PowerShell define the default display of objects in the PowerShell console.
+The `Format.ps1xml` files in PowerShell define the default display of objects in the PowerShell console. You can create your own `Format.ps1xml` files to change the display of objects or to define default displays for new object types that you create in PowerShell.
 
 ### [about_Function_Provider](about_Function_Provider.md)
-
-Provides access to the PowerShell Functions.
+Describes the features of the Function provider.
 
 ### [about_Functions](about_Functions.md)
-
 Describes how to create and use functions in PowerShell.
 
 ### [about_Functions_Advanced](about_Functions_Advanced.md)
-
 Introduces advanced functions that act similar to cmdlets.
 
 ### [about_Functions_Advanced_Methods](about_Functions_Advanced_Methods.md)
-
 Describes how functions that specify the `CmdletBinding` attribute can use the methods and properties that are available to compiled cmdlets.
 
 ### [about_Functions_Advanced_Parameters](about_Functions_Advanced_Parameters.md)
-
 Explains how to add parameters to advanced functions.
 
 ### [about_Functions_CmdletBindingAttribute](about_Functions_CmdletBindingAttribute.md)
-
 Describes the attribute that makes a function work like a compiled cmdlet.
 
 ### [about_Functions_OutputTypeAttribute](about_Functions_OutputTypeAttribute.md)
-
 Describes an attribute that reports the type of object that the function returns.
 
 ### [about_Group_Policy_Settings](about_Group_Policy_Settings.md)
-
-Describes the Group Policy settings for Windows PowerShell
+Describes the Group Policy settings for PowerShell
```

Comment:
Missing a period.  This is inconsistent in a few places, please check the whole doc to make it consistent.